firestore/angular-rewrite/firestore.rules

```diff
@@ -2,17 +2,14 @@ service cloud.firestore {
   match /databases/{database}/documents {
     match /restaurants/{restaurant} {
       match /ratings/{rating} {
-        allow read: if request.auth != null;
-        allow write: if request.auth.uid == request.resource.data.userId;
+        // Allow anyone to read, but only auth'd users to write
+        allow read: if request.time > timestamp.date(2020, 1, 1);
+        allow write: if request.auth != null;
       }
 
-      allow read: if request.auth != null;
+      allow read: if request.time > timestamp.date(2020, 1, 1);
```

Comment:
What are we testing with the timestamp date?

Reply:
I wanted a way to ensure that unauthenticated users would always be able to read (but not write!) items in Firestore. Changed to `allow read: if true` in an upcoming commit